### PR TITLE
Add blog post: Deno 2.0 and what it means for Appwrite Functions

### PR DESCRIPTION
**Pull request by ebenezerdon**
## What does this PR do?

- Adds Deno 2.0 blog post
- Add blog post assets

## Test Plan

- Run the website and navigate to `/blog/post/deno-2-appwrite-functions`
- Confirm that blog post exists and that the assets are displayed correctly

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/routes/blog/post/deno-2-appwrite-functions/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+Deno was built to address most of Node.js' issues, especially around security, dependency management, and TypeScript support. Deno is designed to be a more secure and modern alternative to Node.js, with a focus on server-side applications, which makes it a good choice for Appwrite Functions.
+
+This article will give you an overview of Deno 2.0, its benefits, key features, and how it compares to Node.js in Appwrite.
+
+# Key features of Deno 2.0
+
+Deno 2.0 builds on the foundation laid by the earlier versions, introducing several new features and improvements. Here are some of the key features of Deno 2.0:
+
+## 1. Security by default
```
Comment (adityaoberai) [inline src/routes/blog/post/deno-2-appwrite-functions/+page.markdoc:22]:
```suggestion
## 1. Secure by default
```

Just double check what fits better here

File: src/routes/blog/post/deno-2-appwrite-functions/+page.markdoc [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+4. **Migration**: Migrating existing Node.js projects to Deno can be complex, especially for large codebases with many dependencies. While Deno 2.0 has improved compatibility with Node.js, the process of migrating code can still be time-consuming and error-prone.
+
+Despite these challenges, Deno 2.0 looks compelling and will likely gain more traction as more developers become familiar with its features and benefits.
+
+# Appwrite's support for Deno 2.0
+
+Deno 2.0 just released today, and Appwrite is already ahead of the curve in supporting it. In Appwrite, you can now build serverless functions using Deno, and take advantage of its security features, TypeScript support, and modern module management. To get started with Deno in Appwrite, head over to the [Appwrite console](https://cloud.appwrite.io/), and just like you would with Node.js when creating a new function, select Deno as the runtime.
+
+We'll be releasing a detailed tutorial on building serverless functions with Deno and Appwrite soon, so stay connected with us on [Discord](https://appwrite.io/discord) and [Twitter](https://twitter.com/appwrite_io) to see it first.
```
Comment (adityaoberai) [inline src/routes/blog/post/deno-2-appwrite-functions/+page.markdoc:112]:
```suggestion
We will release a detailed tutorial on building serverless functions with Deno and Appwrite soon, so stay connected with us on [Discord](https://appwrite.io/discord) and [Twitter](https://twitter.com/appwrite) to see it first.
```

**Review by adityaoberai — CHANGES_REQUESTED** [review]
@ebenezerdon couple of changes here

File: src/routes/changelog/(entries)/2024-10-09.markdoc [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+cover: /images/changelog/2024-10-09.png
+---
+
+Today, Appwrite expands the Cloud Function ecosystem with a new, powerful runtime that offers developers simplicity and security—introducing Deno support. 
+
+With [Deno](https://deno.com/), you can write modern, fast, and secure cloud functions in JavaScript and TypeScript, which includes all-in-one tooling such as a [linter](https://docs.deno.com/runtime/reference/cli/linter/), [test runner](https://docs.deno.com/runtime/reference/cli/test/), [formatter](https://docs.deno.com/runtime/reference/cli/formatter/) and more, as well as [web standards API support](https://docs.deno.com/runtime/reference/web_platform_apis/).
+
+{% arrow_link href="/blog/post/deno-runtime-announcment" %}
+Read the announcement to learn more
+{% /arrow_link %}
```
Comment (adityaoberai) [inline src/routes/changelog/(entries)/2024-10-09.markdoc:15]: We have this changelog in #1417 also?

Comment (adityaoberai) [reply src/routes/changelog/(entries)/2024-10-09.markdoc:15]: @ebenezerdon

Comment (adityaoberai) [reply src/routes/changelog/(entries)/2024-10-09.markdoc:15]: imho, this links to the blog in #1417 and it makes more sense to have it in the other PR

Comment (LauraDuRy) [reply src/routes/changelog/(entries)/2024-10-09.markdoc:15]: i don;t see it there. and we don't have time. they both will go live anyway

Comment (Meldiron) [reply src/routes/changelog/(entries)/2024-10-09.markdoc:15]: All good, all goes together

**Pull request merged**